src/pages/Assets/AssetDetailsPage.vue

```diff
@@ -0,0 +1,226 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+import { computed, onMounted, ref } from 'vue';
+import { http } from '@/shared/api';
+import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
+import { useTable } from '@/shared/lib/table';
+import BaseTable from '@/shared/ui/components/BaseTable.vue';
+import { type filterTransactionsModel as ftm, TransactionStatusFilter } from '@/features/filter-transactions';
+import BaseHash from '@/shared/ui/components/BaseHash.vue';
+import { transactionModel } from '@/entities/transaction';
+import { useWindowSize } from '@vueuse/core';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import BaseTabs from '@/shared/ui/components/BaseTabs.vue';
+import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
+import DataField from '@/shared/ui/components/DataField.vue';
+import type { TabAssetsSort } from '@/features/filter-transactions/model';
+import { sortOptions } from '@/features/filter-transactions/model';
+import { elapsed } from '@/shared/lib/time';
+
+const router = useRouter();
+
+const { handleUnknownError } = useErrorHandlers();
+
+const HASH_BREAKPOINT = 800;
+const { width } = useWindowSize();
+
+const hashType = computed(() => (width.value < HASH_BREAKPOINT ? 'medium' : 'full'));
+
+const assetId = computed(() => {
+  const assetName = router.currentRoute.value.params['id'];
+
+  const domain = router.currentRoute.value.hash;
+
+  if (typeof assetName === 'string') return assetName + domain;
+
+  return assetName[0] + domain;
+});
+
+const asset = ref<AssetDefinition | null>(null);
+const isFetchingAsset = ref(false);
+
+onMounted(async () => {
+  try {
+    isFetchingAsset.value = true;
+    asset.value = await http.fetchAssetDefinition(encodeURIComponent(assetId.value));
+
+    await transactionsTable.fetch();
+  } catch (e) {
+    handleUnknownError(e);
+  } finally {
+    isFetchingAsset.value = false;
+  }
+});
+
+const transactionStatus = ref<ftm.Status>(null);
+const transactionType = ref<TabAssetsSort>('recent');
+
+// use asset's transactions from payload instead or replace with fetching asset's transactions method
```

Comment:
I don't understand this comment. Could you rephrase?

And, should it be a TODO/FIXME?